github/repos_community_health_test.go

```diff
@@ -0,0 +1,85 @@
+// Copyright 2017 The go-github AUTHORS. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package github
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"reflect"
+	"testing"
+	"time"
+)
+
+func TestRepositoriesService_GetCommunityHealthMetrics(t *testing.T) {
+	setup()
+	defer teardown()
+
+	mux.HandleFunc("/repos/o/r/community/profile", func(w http.ResponseWriter, r *http.Request) {
+		testMethod(t, r, "GET")
+		testHeader(t, r, "Accept", mediaTypeRepositoryCommunityHealthMetricsPreview)
+		fmt.Fprintf(w, `{
+				"health_percentage": 100,
+				"files": {
+					"code_of_conduct": {
+						"name": "Contributor Covenant",
+						"key": "contributor_covenant",
```

Comment:
Why not include `"url": null,` here, just like in the [example](https://developer.github.com/v3/repos/community/#retrieve-community-health-metrics)? Test would still pass, right?